src/torcontrol.cpp

```diff
@@ -28,7 +29,7 @@
 #include <event2/thread.h>
 
 /** Default control port */
-const std::string DEFAULT_TOR_CONTROL = "127.0.0.1:9051";
+const std::string DEFAULT_TOR_CONTROL = LOCALHOST_IPV4 + ":9051";
```

Comment:
Not sure about initialisation order here.

Reply:
> Not sure about initialisation order here.

Was working fine for me the past weeks, but dropped the commit adding the constants to simplify things.